samples/web/content/apprtc/apprtc.py

```diff
@@ -546,5 +579,7 @@ def get(self, room_id):
     ('/register/(\w+)', RegisterPage),
     # TODO(jiayl): Remove support of /room/ when all clients are updated.
     ('/room/(\w+)', RoomPage),
+    ('/room/(\w+)/status', RoomStatusPage),
```

Comment:
We already return the state of the room in /register request. Why do we need a new one?